### PR TITLE
Add run step to deploy latest image to Astro Cloud nightly

### PR DESCRIPTION
**Pull request by pankajkoti**
Add a run step to the existing night deploy job to deploy
latest image on Astro Cloud environment.

### CODE REVIEW

**Review by pankajastro — APPROVED** [review]

File: .circleci/integration-tests/script.sh [hunk 1]
```diff
@@ -102,6 +102,11 @@ if [[ ${DEPLOYMENT_INSTANCE} == "staging" ]]; then
   docker push "${IMAGE_NAME}"
 elif [[ ${DEPLOYMENT_INSTANCE} == "astro-cloud" ]]; then
   IMAGE_NAME=${DOCKER_REGISTRY}/${ORGANIZATION_ID}/${DEPLOYMENT_ID}:ci-${BUILD_NUMBER}
+  echo "values:"
+  echo $DOCKER_REGISTRY
+  echo $ORGANIZATION_ID
+  echo $DEPLOYMENT_ID
+  echo $IMAGE_NAME
```
Comment (pankajkoti) [inline .circleci/integration-tests/script.sh:109]:
```suggestion
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -55,11 +55,11 @@ workflows:
   nightly:
     triggers:
       - schedule:
-          cron: "0 19 * * *"
+          cron: "31 * * * *"
```
Comment (pankajkoti) [inline .circleci/config.yml:58]:
```suggestion
          cron: "0 19 * * *"
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -55,11 +55,11 @@ workflows:
...
           filters:
             branches:
               only:
-                - main
+                - 274-deploy-image-to-AC
```
Comment (pankajkoti) [inline .circleci/config.yml:62]:
```suggestion
                - main
```

File: .circleci/integration-tests/Dockerfile [hunk 4]
```diff
@@ -112,10 +112,10 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
...
 
 # install AWS nuke
-RUN wget --quiet  \
-    "https://github.com/rebuy-de/aws-nuke/releases/download/${AWS_NUKE_VERSION}/aws-nuke-${AWS_NUKE_VERSION}-linux-amd64.tar.gz" \
-    && tar -xzvf aws-nuke-${AWS_NUKE_VERSION}-linux-amd64.tar.gz -C /usr/local/bin \
-    && mv /usr/local/bin/aws-nuke-${AWS_NUKE_VERSION}-linux-amd64 /usr/local/bin/aws-nuke
+#RUN wget --quiet  \
+#    "https://github.com/rebuy-de/aws-nuke/releases/download/${AWS_NUKE_VERSION}/aws-nuke-${AWS_NUKE_VERSION}-linux-amd64.tar.gz" \
+#    && tar -xzvf aws-nuke-${AWS_NUKE_VERSION}-linux-amd64.tar.gz -C /usr/local/bin \
+#    && mv /usr/local/bin/aws-nuke-${AWS_NUKE_VERSION}-linux-amd64 /usr/local/bin/aws-nuke
```
Comment (kaxil) [inline .circleci/integration-tests/Dockerfile:118]: Any reason for commenting this out?

Comment (pankajkoti) [reply .circleci/integration-tests/Dockerfile:118]: I forgot to submit my review . The reason is below:
Commenting this out as the command is failing for images built from runtime images as base image. Need to investigate how to fix this.


Screenshot 2022-05-06 at 8 29 21 PM

Comment (pankajkoti) [reply .circleci/integration-tests/Dockerfile:118]: Commenting this out as the command is failing for images built from runtime images as base image. Need to investigate how to fix this.

Comment (phanikumv) [reply .circleci/integration-tests/Dockerfile:118]: i am checking this issue with aws-nuke

Comment (kaxil) [reply .circleci/integration-tests/Dockerfile:118]: Since https://github.com/astronomer/astronomer-providers/pull/323 is merged, we can uncomment it?

Comment (pankajkoti) [reply .circleci/integration-tests/Dockerfile]: yes, perfect. Thank you @phanikumv & @kaxil

**Review by bharanidharan14 — APPROVED** [review]

**Review by rajaths010494 — APPROVED** [review]

**Review by phanikumv — APPROVED** [review]

**Pull request merged**